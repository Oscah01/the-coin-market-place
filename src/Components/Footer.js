import React from 'react';

const Footer = () => (
  <footer>
    <p>
      &copy; 2023 App By Oscar_Deus Powered by &nbsp;
      <a href="https://www.coingecko.com/en/api/documentation">CoinGecko API</a>
    </p>
  </footer>
);

export default Footer;
