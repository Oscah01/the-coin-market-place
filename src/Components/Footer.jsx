import React from 'react';

const Footer = () => (
  <footer>
    <p>
      &copy; 2022 App By Moriasi. Powered by &nbsp;
      <a href="https://www.coingecko.com/en/api/documentation">CoinGecko API</a>
    </p>
  </footer>
);

export default Footer;
