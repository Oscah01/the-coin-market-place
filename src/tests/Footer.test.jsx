import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configureStore';
import Footer from '../Components/Footer';

describe('Footer', () => {
  test('should render the Footer component', () => {
    render(
      <Provider store={store}>
        <BrowserRouter>
          <Footer />
        </BrowserRouter>
      </Provider>,
    );
    expect(Footer).toMatchSnapshot();
  });
});
