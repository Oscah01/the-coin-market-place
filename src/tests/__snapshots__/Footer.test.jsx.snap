// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the Footer component 1`] = `[Function]`;
